nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```diff
@@ -271,6 +331,23 @@ in {
         }
       ]) (builtins.attrNames cfg.extraFiles);
 
+    warnings = lib.optional cfg.secureBoot.enable
```

Comment:
Yes, you don't use this thing and you keep using out of tree lanzaboote. :)

Reply:
No, this is super annoying and I don't want to patch this out out of tree like I already did with the secureboot warning. Maybe do the setting internal so that only people that know, know how to disable it but showing a none actionable warning for every rebuild is just annoying.

Reply:
Wait until the Nixpkgs problem RFC is implemented then, I offered you alternatives already.

Reply:
Right now warnings displayed are usually actionable. This isn't actionable and teaching people that warnings can be ignored. We shouldn't do that.